mrmustard/math/parameter_set.py

```diff
@@ -217,6 +220,111 @@ def __getitem__(self, items: int | Sequence[int]):
 
         return ret
 
+    def _format_value(self, param: Constant | Variable) -> tuple[str, str]:
+        r"""Format parameter value and shape strings."""
+        try:
+            value = math.asnumpy(param.value)
+        except (ValueError, TypeError, AttributeError):
+            return "array", str(getattr(param.value, "shape", "scalar"))
```

Comment:
What kind of object do you imagine leading to these exceptions @ziofil? `Constant`s and `Variable`s have the `@property` `value`, and so I don't see how an `AttributeError` would ever be raised. Even if it were, a second `AttributeError` would be raised when trying to access its "shape" if it didn't exist itself. As for a `ValueError` or `TypeError`, it'd have to be something that couldn't be converted with `math.asnumpy`, but I can't think off the top of my head what would fail.